docs/Platform-Architecture.md

```diff
@@ -6,17 +6,33 @@ The user-facing component of the OpenWPM platform is the Task Manager. The Task
 
 ## Instantiating a Task Manager
 
-All automation code is contained within the `openwpm` folder; the Task Manager code is contained in `openwpm/TaskManager.py`.
+All automation code is contained within the `openwpm` folder; the Task Manager code is contained in `openwpm/task_manager.py`.
 
 Task Managers can be instantiated in the following way:
+```python
+from opemwpm.task_manager import TaskManager
+from openwpm.config import (
+    BrowserParams,
+    ManagerParams,
+)
 
-`manager = TaskManager.TaskManager(manager_params, browser_params)`
+number_of_browser = 5    #  Number of browsers to spawn
+
+# Instantiating Browser and Manager Params with default values.
+manager_params = ManagerParams(num_browsers = number_of_browsers)
+browser_params = [BrowserParams() for bp in range(manager_params.num_browsers)]
+
+# These instances can be used to modify default values of both browser and manager params.
+manager_params.memory_watchdog = True
+manager_params.process_watchdog = True
```

Comment:
Could you use setting the the output directory as an example instead of this?
People that are on MacOS and just blindly copy and paste the code might otherwise run into exceptions, which we don't want.

Reply:
For sure!...